libs/api/domains/auth-admin/src/lib/models/tenant.model.ts

```diff
@@ -0,0 +1,12 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+import { TenantEnvironment } from './tenant-environment.model'
+
+@ObjectType('AuthAdminTenant')
+export class Tenant {
+  @Field()
+  id!: string
+
+  @Field(() => [TenantEnvironment])
+  environments!: TenantEnvironment[]
```

Comment:
~We could have a "defaultEnvironment" field resolver on all of our models which just resolves the first (or last) environment to simplify list queries. 

Then list queries would only query `defaultEnvironment` properties and not receive duplicate displayName, applicationsCount etc for each environment.

Not sure if the "max(applicationsCount)" is important enough to warrant querying all environments, or if we should move it to the server somehow, ie `tenant.defaultEnvironment.applicationsCount = Math.max(tenant.environments.map(e => e.applicationsCount)`. Or call it something else like `tenant.mergedEnvironment`?

Reply:
Yes that is a great suggestion.